File: 1.FDP/2° Competencia/C++/Basicos/Ejercicio5.cpp
#include <iostream>
using namespace std;

int main(){
	cout << "Convertir pesos a dolares.\n";
	double Pesos, ValorDolar, Dolares;
	cout << "Cantidad de pesos: ";
	cin >> Pesos;
	cout << "Valor del dolar: ";
	cin >> ValorDolar;
	Dolares= Pesos/ValorDolar;
	cout << "Cantidad de dolares = " << Dolares << "\n";
	return 0;
}

File: 1.FDP/2° Competencia/Java/Basicos/Ejercicio5.java
import java.util.Scanner;

public class Ejercicio5 {
	public static void main (String[] args){
		Scanner scan= new Scanner(System.in);
		//
		System.out.println("Calcular la convercion de pesos a dolares.\n");
		double Pesos, Dolares, Convercion;
		System.out.println("Pesos: ");
		Pesos= scan.nextDouble();
		System.out.println("Valor del dolar: ");
		Dolares= scan.nextDouble();
		Convercion= Pesos/Dolares;
		System.out.println("Cantidad de dolares= " + Convercion);
		//
	}
}